src/explorer/index.ts

```diff
@@ -81,7 +81,7 @@ function handleNodeDblClick(node: Node, e: MouseEvent) {
   if (clickedNode == null) {
     return
   }
-  if (clickedNode.type === NodeType.FOLDER) {
+  if (clickedNode.type !== NodeType.FILE) {
```

Comment:
this becomes problematic, when we introduce more types than we need. I think that specifying that only Folders are open-able is more semantic and less likely to bite us in the behind